packages/synapse-react-client/src/theme/DefaultTheme.tsx

```diff
@@ -10,14 +23,53 @@ export const defaultMuiThemeOptions: ThemeOptions = {
   palette: palette,
   components: {
     MuiAlert: {
+      defaultProps: {
+        square: true,
+        iconMapping: {
+          error: <HighlightOffTwoTone />,
+          info: <InfoTwoTone />,
+          success: <CheckCircleTwoTone />,
+          warning: <ErrorTwoTone />,
+        },
+      },
       styleOverrides: {
         root: ({ ownerState, theme }) => ({
-          borderRadius: '0px',
-          fontSize: '16px',
+          alignItems: 'center',
+          fontSize: '14px',
           borderLeft: `10px solid ${
             // The default severity is 'success'.
             theme.palette[ownerState.severity || 'success']['main']
           }`,
+          color: theme.palette.grey[900],
+        }),
+        icon: ({ theme }) => ({
+          marginRight: theme.spacing(3),
+        }),
+        action: ({ ownerState, theme }) => ({
+          paddingRight: theme.spacing(2),
+          ...(ownerState.onClose &&
+            !ownerState.action && { color: theme.palette.grey[700] }),
+        }),
+        standardSuccess: ({ theme }) => ({
+          backgroundColor: lighten(theme.palette.success.light, 0.8),
+        }),
+        standardError: ({ theme }) => ({
+          backgroundColor: lighten(theme.palette.error.light, 0.8),
+        }),
+        standardWarning: ({ theme }) => ({
+          backgroundColor: lighten(theme.palette.warning.light, 0.8),
+        }),
+        standardInfo: ({ theme }) => ({
+          backgroundColor: lighten(theme.palette.info.light, 0.8),
+        }),
```

Comment:
Adjust the background color to more closely match the [design](https://www.figma.com/file/0oPm5lLSUva8kyfVNMS6FA/Sage-Style-%26-Component-Library?type=design&node-id=3421-33709&t=Wxokkj1ssR5jpNon-4) using the same process as [MUI](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Alert/Alert.js#L60) to adjust the color. Color is still slightly different than the design -- happy to hardcode the exact colors or adjust the theme in another way, if there's a better approach!

Reply:
This is probably fine. I didn't even notice a difference until I got here